wal/util.go

```diff
@@ -72,20 +72,21 @@ func readWalNames(dirpath string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	wnames := checkWalNames(names)
+	badf := func(f string) { plog.Warningf("ignored file %v in wal", f) }
```

Comment:
this is so that it doesn't print out `ignored` warnings when checking for existence; it's preserving previous output behavior